bin/workbench-daemon.sh

```diff
@@ -88,6 +89,23 @@ function wait_for_workbench_to_die() {
   fi
 }
 
+function check_jdbc_jar() {
+  if [[ -d "${1}" ]]; then
+    mysql_connector_exists=$(find -L "${1}" -name "mysql-connector*")
+    if [[ -z "${mysql_connector_exists}" ]]; then
+      if [[ ${GET_MYSQL_JAR} = true ]]; then
+        download_mysql_jdbc_jar
+      else
+        echo -e "\\033[31mError: There is no mysql jdbc jar in workbench/lib.\\033[0m"
+        echo -e "\\033[31mPlease download a mysql jdbc jar and put it under workbench/lib manually.\\033[0m"
+        echo -e "\\033[31mOr add a parameter getMysqlJar, like this:\nworkbench-daemon.sh start getMysqlJar\\033[0m"
```

Comment:
Change to 
`like this:\n./bin/workbench-daemon.sh start getMysqlJar ??`
or
`like this:\n./workbench-daemon.sh start getMysqlJar ??`

Reply:
Both /bin/workbench-daemon.sh and ./workbench-daemon.sh will work. 
Just use ./bin/workbench-daemon.sh just like the docs.